### ISSUE TITLE
ANSWER THE QUESTIONS IN COMMENTS ON THIS ISSUE

### ISSUE THREAD

**Issue by fredlabbe** [fredlabbe/Prototype-2-cart315#1]

**Comment by fredlabbe** [comment]
Just write a comment here!

**Comment by LeVieuxSinge** [comment]
**Did you find your way easily?**
Yes, the environment design was well executed. The path was clear and the outline of the trees and foliage helped marked the way.

**What was the atmosphere in the prototype?**
The wind sounds felt like I was at the beach. The lighting was nice, the night ambient was well done. 

**What is the story in the small game in your opinion?**
The story of big foot lost in the wood and trying to reach for help. Unfortunately for him, people are scared and are hiding from him.

**How did you feel as the character entering the city?**
The camera kind of broke the immersion for me as it was very hard to control. Otherwise, I felt like my character was out of place in the city.

**Were you a good or a bad guy in your opinion?**
I'm the bad guy (duh !)

**Would this idea for a game be worth pursuing?**
Have you ever played Walking Simulator because I think your game as similar goals.

**Does playing as a monster-like character make seem for players that the main character is a bad guy or not?**
Monsters are always associated to being bad.

**Comment by PatrickChavez** [comment]
**NOTE: I was having trouble with the camera during playtesting: everytime the character walked, the camera would shake uncontrollably. This made playtesting difficult, so I decided to use a first person controller for the playtest session, rather than the third person one.** 

**I will try to answer the questions to the best of my ability.**

**1. Did you find your way easily?**
Yes, getting to the village was easy due to the mostly linear path and the “mini fortress checkpoint” guiding me.

**2. What was the atmosphere in the prototype?**
It had an eerie fantasy atmosphere. The fact that you play as a monster heading to a village in the dead of night helps to set an ominous tone.

**3. What is the story in the small game in your opinion?**
You play as a monster heading to a nearby village in order to fulfill some unknown goal: perhaps it is to eat the villagers or perhaps the monster was once human and is looking for a way to turn back to normal?

**4. How did you feel as the character entering the city?**
I could not play as the monster for this part.

**5. Were you a good or a bad guy in your opinion?**
It is hard to tell: knowing the morality of the monster depends on its motivations and actions during the game, both of which are unknown to me.

**6. Would this idea for a game be worth pursuing?**
Yes, it has a good atmosphere and has the interesting premise of playing as a monster. There was a lot of care put into the environments and it would be interesting to see how far this prototype could go.

**Comment by sebchampoux** [comment]
> Did you find your way easily?

Yes, the path was easy to follow and eventually brought me directly to the village.

> What was the atmosphere in the prototype?

The atmosphere was eerie and slightly scary.  It establishes the mood very well.

> What is the story in the small game in your opinion?

Good question, it looks like a small poor village in distress.  I'm not a very good conceptualizer so I won't try to guess any further though :).  I don't know if you have seen the Meeseeks episode from Rick and Morty, but your prototype made me think of that for some reason, the village where Rick and Morty go and they have to get money from the giant at the top of the beanstalk.

> How did you feel as the character entering the city?

I was relieved to finally reach it as the running speed of my character was slow.  Seeing the reaction of the villagers upon my arrival would have told me a lot.

> Were you a good or a bad guy in your opinion?

I'm not too sure.  The 3D model of the character doesn't look like a good guy.  Seeing the villagers' response upon my arrival would have been helpful here as well.

> Would this idea for a game be worth pursuing?

Yeah definitely!  It's a great prototype and if you pick this one as your final project, I'll be looking forward to the final version.

### Other unrelated feedback
I found the controls difficult to use.  The mouse was very sensitive which made it difficult to aim as I wished, and keep my view stable as well.  The character also ran very slowly, which was a bit annoying considering the size of the map.  However, these are minor details that I think could easily be fixed if you were to push this further.  Just wanted to share so it could be improved upon if applicable.

**Comment by CocoChao** [comment]
**- Did you find your way easily?**
Yes, it was very intuitive, and the path was clear.

**- What was the atmosphere in the prototype?**
It was very mysterious, as if I was the only character alive and I was revisiting old, abandoned villages.

**- From what you understand, what is the story in the small game?**
To find my way back to my village or to reach the nearest village.

**- How did you feel as the character entering the city?**
It felt like it was the character’s natural habitat and there was nothing to be afraid of.

**- From what you understand, were you a good a bad guy?**
I assume that I was a bad guy from my character’s physique and the ambience created by the environment. However, it would be interesting to see what sort of interactions my character can do if he turns out to be harmless. 

**- Would this idea for a game be worth pursuing?**
Yes, the game already has a lot of potential already. You included many elements in your game such as a story (text), environment, sounds, sky and lighting. However, there are some bugs that I encountered while playing your game. Sometimes it looks like the character is floating, the shadows from the trees glitches at the beginning, and the mouse was a little disorienting (it was hard to return to the main camera view). I suggest you add more challenges/interactions or people to meet on the way to the village to satisfy the player.
